src/store/modules/layers.store.js

```diff
@@ -527,6 +556,11 @@ const mutations = {
             timedLayer.timeConfig.getTimeEntryForYear(year)
         )
     },
+    setLegacyLayerYear(state, { timedLayer, year }) {
+        timedLayer.timeConfig.updateCurrentTimeEntry(
+            timedLayer.timeConfig.getTimeEntryForYear(year)
+        )
+    },
```

Comment:
`setLayerYear` checks the store for active layers using the layer Id,

In the case of layers that are not active by default (for example: swissimage), this means we can't find them in the store and it will refuse to update their time series. A better name would have been `setNonActiveLayerYear`

Reply:
Active layer means that they are in the layer list, but not necessarily visible